buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -106,15 +106,19 @@ internal class YamlWriter {
   }
 
   @VisibleForTesting
-  internal fun createDeviceString(devices: List<Device>): String = buildString {
+  internal fun createDeviceString(devices: List<Map<String, String?>>): String = buildString {
     appendln("  device:")
     for (device in devices) {
-      appendln("  - model: ${device.model}")
-      appendln("    version: ${device.version}")
-      device.orientation?.let {
+      val model = device["model"]
```

Comment:
```suggestion
      val model = device["model"]!!
```
I believe `model` and `version` are required fields. Rather than printing `- model: null`, let's fail fast here.
Maybe we can write a test for this case too.

Reply:
Good call, will fix this 👍

Reply:
Alright, I added a `RequiredDeviceKeyMissingException` and throw that when "model" or "version" are `null`.